docker-compose.yml

```diff
@@ -1,31 +1,35 @@
 version: '3.1'
 
 services:
-
   server:
     image: deviavir/zenbot:unstable
+    # build: .
     volumes:
-      - ./conf.js:/app/conf.js
-      - ./extensions:/app/extensions
-      - ./simulations:/app/simulations
+      - '.:/app'
```

Comment:
I don't think this is a good idea, we're installing `node_modules` in the Dockerfile and this looks like it overwrites that

Reply:
Ah, yes, you're right. I'll come up with a different solution.